.tool-versions

```diff
@@ -1,2 +1,2 @@
 nodejs 18.17.0
-python 3.11.1
+python 3.13.1
```

Comment:
Aren't we on 3.11?

Reply:
Eh. That's only relevant for running the CDK tooling. It has no bearing on what runs in the container. I've already got a follow-on PR to update the docker container to 13 in the works.

Reply:
Just pushed a new commit that brings the Docker image to Python 3.13.